File: tests/typesafety/python/exercise_status_start.py
# expect: passes
# title: The typo makes a branch unreachable
def next_action(status: str) -> str:
    if status == "recieved":  # typo: never matches "received"
        return "start picking"
    if status == "picked":
        return "pack the box"
    if status == "shipped":
        return "send the tracking mail"
    return "unknown status"

File: tests/typesafety/typescript/exercise_status_start.ts
// expect: passes

export function nextAction(status: string): string {
  if (status === "recieved") {
    return "start picking";
  }
  if (status === "picked") {
    return "pack the box";
  }
  if (status === "shipped") {
    return "send the tracking mail";
  }
  return "unknown status";
}
